### PR TITLE
Fix get started button positioning on small screens for v4 clusters

### PR DESCRIPTION
**Pull request by axbarsan**
Fixes giantswarm/giantswarm#10119

## Preview

**Same scenario as in the issue**

image

**Even smaller screens**
* URI bar is scrollable horizontally by sliding with the finger

image

### CODE REVIEW

**Comment by axbarsan** [comment]
> LGTM!

too early man 😬

**Comment by oponder** [comment]
Automatic mode...

File: src/components/Cluster/ClusterDetail/V4ClusterDetailTable.js [hunk 1]
```diff
@@ -31,6 +31,47 @@ const WrapperDiv = styled.div`
...
+  & > div:nth-of-type(2) {
+    margin-left: 0;
+    margin-right: 0;
+
+    ${mq(CSSBreakpoints.Large)} {
+      & > * {
+        margin-left: 0;
+      }
+    }
+  }
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/V4ClusterDetailTable.js:58]: Unfortunately I have to do things like this to override the high specificity in the not-so-extendable `FlexRowWithTwoBlocksOnEdges`.
I vote for refactoring this for the new cluster detail view screen

Comment (oponder) [reply src/components/Cluster/ClusterDetail/V4ClusterDetailTable.js:58]: Vote accepted!

**Pull request merged**